llvm/include/llvm/Support/RISCVTargetParser.def

```diff
@@ -19,6 +19,7 @@ PROC(SIFIVE_S54, {"sifive-s54"}, FK_64BIT, {"rv64gc"})
 PROC(SIFIVE_S76, {"sifive-s76"}, FK_64BIT, {"rv64gc"})
 PROC(SIFIVE_U54, {"sifive-u54"}, FK_64BIT, {"rv64gc"})
 PROC(SIFIVE_U74, {"sifive-u74"}, FK_64BIT, {"rv64gc"})
+PROC(CV32E40P, {"CV32E40P"}, FK_NONE, {"rv32imfc"})
```

Comment:
Two minor tweaks here, I'm not 100% sure on the exact answer for these, but interested in comments.

1. This list looks like it was sorted before adding this, so should cv32e40p be moved up the list. (Subpoint, should generic be top of the list regardless?)
2. For consistency, I think perhaps cv32e40p might want to be made lowercase.

Reply:
Regarding the order in RISCVTargetParser.def, i can move the list, but i think keeping the generic targets at the top makes sense.
Also, this file is already removed in upstream llvm since this [commit](https://github.com/llvm/llvm-project/commit/ac1ffd3caca12c254e0b8c847aa8ce8e51b6cfbf) as they moved to a tablegen based generation for the cpu definitions.

Regarding 2.: Lowercase probably makes more sense, i will update it.